docs/7.x/pack.md

```diff
@@ -462,6 +462,13 @@ nodeProfiles:
           # Recursive defines a recursive mount, i.e. all submounts under specified path
           # are also mounted at the corresponding location in the targetPath subtree
           recursive: false
+          # SELinux label to use for the mount.
+          # If a label is given, the installer will label the directory
+          # using the specified label.
+          # If left empty or unspecified, the installer will label the directory
+          # using the default file label.
+          # Special label value of "none" explicitly turns off labeling for the directory
+          label: "SELinux label"
```

Comment:
I didn't notice this when reviewing the code, but I think that "label" might be a too generic property of a node profile is this is SELinux specific. Maybe rename it to "seLinuxLabel" to make it explicit?